internal/api/v1beta1/role.go

```diff
@@ -81,6 +81,49 @@ func (h Handler) ListRoles(ctx context.Context, request *shieldv1beta1.ListRoles
 	return &shieldv1beta1.ListRolesResponse{Roles: roles}, nil
 }
 
+func (h Handler) CreateRole(ctx context.Context, request *shieldv1beta1.CreateRoleRequest) (*shieldv1beta1.CreateRoleResponse, error) {
```

Comment:
We need to protect this endpoint in `interceptor`

Reply:
it was already there in interceptor before this